mq/main/packager-opensource/pom.xml

```diff
@@ -800,9 +800,8 @@
             <artifactId>jakarta.servlet-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>jakarta.transaction</groupId>
-            <artifactId>jakarta.transaction-api</artifactId>
-            <version>1.3.1</version>
+            <groupId>org.glassfish.main.transaction</groupId>
```

Comment:
Shouldn't this just remain jakarta.*?

Reply:
It's the same dependency as in 
https://github.com/eclipse-ee4j/openmq/blob/edbf0f5438bf9764d781f6b058cc1e8347c7d092/mq/main/bridge/bridge-jms/pom.xml#L83-L87
which itself depends on `jakarta.transaction-api`.
As `jakarta.transaction-api` is not directly referenced by any module I thought it will be better to not do it here neither. Thus using managed dependency defined for `jta` in 
https://github.com/eclipse-ee4j/openmq/blob/edbf0f5438bf9764d781f6b058cc1e8347c7d092/mq/main/pom.xml#L681-L685

Reply:
It (jta) would not be present in final zip, as no ant copy rule picks it. But as noted above - this dep is not needed for packager. So I'm moving branch back one commit (as change for dependency is not needed for 456). Sorry for noise about jta.